### PR TITLE
Tomhaile/ch16690/helpers to start pop geth should use file

### PR DESCRIPTION
**Pull request by bthaile**
Add ability to create pop docker images. For now have to run `yarn docker:build` in augur-tools dir. Not sure why it stalls when run from root.

Added helper to run augur-core docker image (needed for creating pop dockers) `yarn docker:augur-core`.
Added helper to run augur docker image (needed for creating pop dockers) `yarn docker:augur`

### CODE REVIEW

File: packages/augur-node/package.json [hunk 1]
```diff
@@ -66,7 +66,7 @@
     "@types/uuid": "3.4.4",
     "@types/ws": "3.0.2",
     "async": "2.6.1",
-    "augur.js": "6.0.0-0",
+    "augur.js": "6.0.1",
```
Comment (justinbarry) [inline packages/augur-node/package.json:69]: I changed the version of augur.js to "6.0.0-0" presumably to fix the same issue you had. We should probably move it to "6.0.1" but I leave that to our release master @petong.

File: packages/augur-core/package.json [hunk 2]
```diff
@@ -76,14 +76,14 @@
...
     "@types/yargs": "11.0.0",
+    "@velma/ganache-core": "2.1.0-beta.0-velma-1.0.0-rc.2",
     "async-file": "2.0.2",
     "async-mkdirp": "1.2.7",
     "async-parallel": "1.2.3",
     "binascii": "0.0.2",
     "bn.js": "4.11.8",
     "copy-dir": "0.3.0",
     "crypto-promise": "2.0.0",
-    "@velma/ganache-core": "2.1.0-beta.0-velma-1.0.0-rc.2",
```
Comment (justinbarry) [inline packages/augur-core/package.json]: Is it outside the scope of this ticket to upgrade this?

File: packages/augur-core/package.json [hunk 3]
```diff
@@ -93,6 +93,7 @@
     "ethjs-signer": "0.1.1",
     "fs-readfile-promise": "3.0.0",
     "get-port": "3.2.0",
+    "npx": "^10.2.0",
```
Comment (justinbarry) [inline packages/augur-core/package.json]: I believe this comes with the system npm install.

File: packages/augur-tools/package.json [hunk 4]
```diff
@@ -4,7 +4,16 @@
   "description": "Tools and dev dependencies",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```
Comment (justinbarry) [inline packages/augur-tools/package.json:7]: @nuevoalex Will this break windows support?

File: packages/augur-tools/scripts/test-pause-unpause-rep.js [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+          var testLegacyReputationTokenTransferPayload = {
+            _to: LEGACY_REPUTATION_TOKEN_TEST_TRANSFER_RECIPIENT,
+            _value: "0x1",
+            meta: auth,
+            tx: { to: legacyReputationToken[networkId].contractAddress }
+          };
+          console.log(
+            chalk.blue.bold("\nPre-pause LegacyReputationToken.transfer...")
+          );
+          augur.api.LegacyReputationToken.transfer(
```
Comment (justinbarry) [inline packages/augur-tools/scripts/test-pause-unpause-rep.js:74]: Some of the contract calls are now promisified. Can this be cleaned up with async/await?

**Review by justinbarry — CHANGES_REQUESTED** [review]
Left some questions for other team members. Wondering if we can leverage lerna to handle to build all vs using a separate script to do it.

Looks like everything thing is there. Way to do the heavy lifting!

File: packages/augur-core/package.json (hunk 2 above)
Comment (bthaile) [reply packages/augur-core/package.json]: Couldnt get lerna to work. It just hangs. Reason for the script

File: packages/augur-tools/scripts/test-pause-unpause-rep.js (hunk 5 above)
Comment (bthaile) [reply packages/augur-tools/scripts/test-pause-unpause-rep.js:74]: Was this merged in? I didnt write this

File: packages/augur-node/package.json (hunk 1 above)
Comment (bthaile) [reply packages/augur-node/package.json:69]: Augur.js version recently got changed. We’ll need to merge thise changes in

File: packages/augur-tools/scripts/test-pause-unpause-rep.js (hunk 5 above)
Comment (justinbarry) [reply packages/augur-tools/scripts/test-pause-unpause-rep.js:74]: You are right. Showed as a delete/create vs a move. My bad.

File: packages/augur-node/package.json (hunk 1 above)
Comment (justinbarry) [reply packages/augur-node/package.json:69]: I'm all for that. Everything *should* release in lockstep after we rename everything. In a perfect world, the version changes would happen as a separate PR. Obviously not possible here given it broke the dev workflow.

I'll move this to a separate PR.

File: packages/augur-core/package.json (hunk 2 above)
Comment (justinbarry) [reply packages/augur-core/package.json]: @bthaile Whats up with the version change?

File: packages/augur-core/package.json (hunk 3 above)
Comment (bthaile) [reply packages/augur-core/package.json]: remove this lib

File: packages/augur-node/package.json (hunk 1 above)
Comment (bthaile) [reply packages/augur-node/package.json:69]: merged from master, so it's should be good.

File: packages/augur-core/package.json (hunk 2 above)
Comment (bthaile) [reply packages/augur-core/package.json]: this was added back in, g2g.

**Review by justinbarry — APPROVED** [review]

**Comment by justinbarry** [comment]
LGTM

**Pull request merged**